IdeventSQLServerTemplate/dbo/Stored Procedures/spUpdateAddress.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE PROCEDURE [dbo].spUpdateAddress
+	@id int,
+	@street nvarchar(50),
+	@city nvarchar(50),
```

Comment:
Er nvarchar(50) nok til parametrene?

Reply:
Maybe, maybe not :) jeg giver dem lidt morfin